### ISSUE TITLE
About the traning loss

### ISSUE THREAD

**Issue by fuxuliu** [UCSD-AI4H/proteinchat#7]
Thank you for sharing the work. 
May I ask your training loss during training? 
I tried to train the proteinchat in using the data you shared, but the loss did not converge.

Thanks.
